docs/book/known_issues.md

```diff
@@ -12,7 +12,10 @@ This section lists the major known issues with VMware vSphere CSI driver. For co
 - [Devops can modify the volume health status of a PVC manually](#issue_6)
 - [Performance regression in Vanilla Kubernetes 1.17 and 1.18 and Supervisor Cluster 7.0 patch releases](#issue_7)
 - [Migrated Volume Deleted by in-tree vSphere plugin remains on the CNS UI](#issue_8)
-= [CnsRegisterVolume API does not validate if the volume to import is already imported or already present in the supervisor cluster](#issue_9)
+- [CnsRegisterVolume API does not validate if the volume to import is already imported or already present in the supervisor cluster](#issue_9)
+- [Supervisor Cluster PVC expanded size is lost and cannot be reclaimed in vSphere 7.0U1 patch release](#issue_10)
```

Comment:
SV devops can manually expand the PVC in SV namespace. Can we document that this is not a supported use case  and is a current product limitation?

Reply:
Sure, I can add it as another Known Issue.

Reply:
@SandeepPissay We have already stated that Offline volume expansion is not supported in Supervisor cluster - https://vsphere-csi-driver.sigs.k8s.io/supported_features_matrix.html. Why do we need to add this as a known issue?

Reply:
Also, SV devops can manually expand a PVC in SV namespace even if the PVC is attached to a PodVM, should we mention that too?